CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf

```diff
@@ -0,0 +1,348 @@
+        <trans-unit id="TemplateDoubleHelp" translate="yes" xml:space="preserve">
+          <source>Please enter a number{? between {2:F1} and {3:F1}}{?, {0}}{?, {1}}.</source>
+          <target state="signed-off">Por favor, informe um número{? entre {2:F1} e {3:F1}}{?, {0}}{?, {1}}.</target>
+          <note from="MultilingualBuild" annotates="source" priority="2">0-current choice, 1-no preference, 2-min, 3-max</note>
+        </trans-unit>
+        <trans-unit id="TemplateEnumManyNumberHelp" translate="yes" xml:space="preserve">
+          <source>You can enter one or more numbers {0}-{1} or words from the descriptions. ({2})</source>
+          <target state="signed-off">Você pode informar um ou mais números de {0}-{1} ou palavras das descrições. ({2})</target>
+          <note from="MultilingualBuild" annotates="source" priority="2">0-min, 1-max and 2-enumerated descriptions</note>
+        </trans-unit>
+        <trans-unit id="TemplateEnumManyWordHelp" translate="yes" xml:space="preserve">
+          <source>You can enter in one or more selections from the descriptions. ({2})</source>
+          <target state="signed-off">Você pode informar uma ou mais opções da descrição. ({2})</target>
+          <note from="MultilingualBuild" annotates="source" priority="2">2-enumerated descriptions</note>
+        </trans-unit>
+        <trans-unit id="TemplateEnumOneNumberHelp" translate="yes" xml:space="preserve">
+          <source>You can enter a number {0}-{1} or words from the descriptions. ({2})</source>
+          <target state="signed-off">Você pode informar um número de {0}-{1} ou palavras das descrições. ({2})</target>
+          <note from="MultilingualBuild" annotates="source" priority="2">0-min, 1-max and 2-enumerated descriptions</note>
+        </trans-unit>
+        <trans-unit id="TemplateEnumOneWordHelp" translate="yes" xml:space="preserve">
+          <source>You can enter in any words from the descriptions. ({2})</source>
+          <target state="signed-off">Você pode informar quaisquer palavras das descrições. ({2})</target>
+          <note from="MultilingualBuild" annotates="source" priority="2">2-enumerated descriptions</note>
+        </trans-unit>
+        <trans-unit id="TemplateEnumSelectMany" translate="yes" xml:space="preserve">
+          <source>Please select one or more {&amp;} {||}</source>
+          <target state="signed-off">Por favor, selecione um(a) ou mais {&amp;} {||}</target>
+        </trans-unit>
+        <trans-unit id="TemplateEnumSelectOne" translate="yes" xml:space="preserve">
+          <source>Please select a {&amp;} {||}</source>
+          <target state="signed-off">Por favor, selecione um(a) {&amp;} {||}</target>
+        </trans-unit>
+        <trans-unit id="TemplateFeedback" translate="yes" xml:space="preserve">
+          <source>For {&amp;} I understood {}. {?"{0}" is not an option.}</source>
+          <target state="signed-off">Com {&amp;} eu entendi {}. {?"{0}" não é uma opção.}</target>
```

Comment:
I didn't understand, was it supposed to be _"Com {&amp;} eu entendi **que** {}. {?"{0}" não é uma opção.}'_ ? It sounds strange in this way.